src/main/kotlin/elan/tweaks/thaumcraft/research/frontend/integration/ThaumcraftResearchTweaks.kt

```diff
@@ -35,7 +35,7 @@ object ThaumcraftResearchTweaks {
   lateinit var singletonInitializer: SingletonInitializer
 
   @Mod.EventHandler
-  fun onInit(event: FMLInitializationEvent) {
+  fun onInit(event: FMLLoadCompleteEvent) {
```

Comment:
Tested and confirmed init order issue. Avaria creates it's aspects in `FMLPostInitializationEvent`. While this mod caches aspect tree on normal init. Moving it to latest possible init instead.